src/metrics/metrics.go

```diff
@@ -93,80 +117,146 @@ func populateWaitTimeMetrics(instanceEntity *integration.Entity, connection *con
 	}
 }
 
-func populateCustomMetrics(instanceEntity *integration.Entity, connection *connection.SQLConnection, query string) {
-
-	rows, err := connection.Queryx(query)
-	if err != nil {
-		log.Error("Could not execute custom query: %s", err.Error())
-		return
-	}
-
-	defer func() {
-		_ = rows.Close()
-	}()
+// Execute one or more custom queries
+func populateCustomMetrics(instanceEntity *integration.Entity, connection *connection.SQLConnection, queries []CustomQuery) {
+	// Execute each query in array
+	for _, query := range queries {
```

Comment:
Do we expect customers to want to define a large number of queries? If so, we should probably run these concurrently

Reply:
I don't expect a lot of queries, and keeping CPU usage down taking slower sample time hit is ok, so I think we can push off parallelism